Cargo.lock

```diff
@@ -131,7 +131,7 @@ dependencies = [
  "nom",
  "num-traits",
  "rusticata-macros",
- "thiserror",
+ "thiserror 1.0.69",
```

Comment:
this is **adding** a dependency?

Reply:
Yeah gossip was mostly in the clear I just updated the lockfiles and aligned with `iroh@main`. That dep should go away at some point on it's own once the deps tree shakes it out.